jerry-ext/include/jerryscript-ext/arg.h

```diff
@@ -90,7 +90,61 @@ typedef enum
   JERRYX_ARG_REQUIRED
 } jerryx_arg_optional_t;
 
+/**
+ * Indicates the rounding policy which will be choose to transform a integer.
+ */
+typedef enum __attribute__((packed))
+{
+  JERRYX_ARG_ROUND, /**< round */
+  JERRYX_ARG_FLOOR, /**< floor */
+  JERRYX_ARG_CEIL /**< ceil */
+} jerryx_arg_round_t;
+
+/**
+ * Indicates whether it will throw error when the integer's range is out of the target type.
+ */
+typedef enum __attribute__((packed))
+{
+  JERRYX_ARG_CLAMP,/**< clamp the number when it is out of range */
+  JERRYX_ARG_NO_CLAMP /**< throw a range error */
+} jerryx_arg_clamp_t;
+
+/**
+ * The structure indicates the options used to transform integer argument.
+ * It will be passed into jerryx_arg_t's  extra_info filed.
+ */
+typedef struct
+{
+  jerryx_arg_round_t round; /**< rounding policy */
+  jerryx_arg_clamp_t clamp; /**< clamping policy */
+} jerryx_arg_int_option_t;
```

Comment:
Maybe it makes more sense to add the `packed` to the struct type.
AFAIK `packed` isn't applied to "parent" types.